src/GafferScene/DeleteSets.cpp

```diff
@@ -44,6 +44,29 @@ using namespace IECore;
 using namespace Gaffer;
 using namespace GafferScene;
 
+namespace
+{
+
+const InternedString g_cameras( "__cameras" );
+const InternedString g_lights( "__lights" );
+
+bool keep( InternedString set, const std::string &names, bool invert )
+{
+	if( StringAlgo::matchMultiple( set, names ) != (!invert) )
```

Comment:
Personally, I find this a lot harder to mentally unpick than `StringAlgo::matchMultiple( set, names ) == invert`

Reply:
Switched in 68ddeae